packages/sdk/wallets/wallet-ledger/src/ledger-signer.ts

```diff
@@ -116,7 +116,7 @@ export class LedgerSigner implements Signer {
       )
 
       return {
-        v: parseInt(sig.v, 10),
+        v: parseInt(sig.v, 16),
```

Comment:
the base 10 here seemed like an error TBH

Reply:
hmm  if it worked though....

Reply:
after testing with physical device, it was indeed base 10, weird.